hca_util/command/area.py

```diff
@@ -1,7 +1,7 @@
 from hca_util.local_state import get_local_state, set_local_state, LocalState
 
 
-class CmdDir:
+class CmdArea:
```

Comment:
there's no area command, so i'd move any functions here elsewhere to remove confusion.

Reply:
Ultimately I agree. However, in the interests of getting something out quickly I propose to put off this refactoring.